packages/@webex/plugin-meetings/src/meeting/locusMediaRequest.ts

```diff
@@ -202,6 +204,7 @@ export class LocusMediaRequest extends WebexPlugin {
       correlationId: this.config.correlationId,
       clientMediaPreferences: {
         preferTranscoding: this.config.preferTranscoding,
+        ipver: request.ipVersion,
```

Comment:
should this be `request.ipVersion || unknown` I know it's defaulted at input so maybe this isn't relevant, but it's the only case where the ipversion isnt a config value of the SDK

if we *dont* have this, should we have a 

if (!request?.ipVer) {
// throw UnsupportedParamsException || Logger.warn('need ipver')
}

Reply:
it's a required field of the `Request` and LocusMediaRequest is private to SDK, so typescript should catch if it's missing.